### PR TITLE
 Upgrade for PR #2807 #2823

### PR DESCRIPTION
**Pull request by Intuneteq**
# Fixes Issue/Linear Ticket

PR fixes approach to #2807 
Though the page still doesn't automatically upload when changes are made to user's profile, user needs to refresh so the avatar will updated in message box
( This could be an existing issue or a linear ticket )

**My PR Closes #issue_number_here** or
**My PR Fixes ID-linear_ticket_number_here**

# Changes proposed

## What did you do?

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos
https://drive.google.com/file/d/1u5douG4JWh4uYjDRrHP498lquA1GZNV0/view?usp=drivesdk

### CODE REVIEW

File: packages/utilities/src/workspace.js [hunk 1]
```diff
@@ -9,6 +9,12 @@ import { BASE_API_URL } from "./constants";
  * @description uses the email to get the user's info in the workspace from the local cache or the server
  */
 
+// export const tabsOnCurrentUser = () => {
+//   const { user } = useContext(ProfileContext);
+//   console.log(user)
+//   return { user }
+// }
+
```
Comment (billmal071) [inline packages/utilities/src/workspace.js:17]: remove.

**Review by billmal071 — CHANGES_REQUESTED** [review]

**Comment by Intuneteq** [comment]
Changes Done

File: packages/main/src/components/protected/topbar/components/TopBarSearchModal.jsx [hunk 2]
```diff
@@ -43,6 +43,7 @@ const TopBarSearchModal = () => {
   };
 
   const onSearchSubmit = async e => {
+    console.log("filters", filters);
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/TopBarSearchModal.jsx:46]: Remove this console.log

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: packages/main/src/components/protected/topbar/components/TopBarSearchModal.jsx (hunk 2 above)
Comment (Intuneteq) [reply packages/main/src/components/protected/topbar/components/TopBarSearchModal.jsx]: Fixed

**Review by JulianaSau — APPROVED** [review]

**Pull request merged**